platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/framework/validation/TimestampChecker.java

```diff
@@ -39,10 +39,10 @@ public static void validateConsensusTimestamps(
                                 "Consensus time does not increase!%n"
                                         + "Event %s consOrder:%s consTime:%s%n"
                                         + "Event %s consOrder:%s consTime:%s%n",
-                                previousConsensusEvent.toShortString(),
+                                previousConsensusEvent.getHashedData().getDescriptor(),
                                 previousConsensusEvent.getConsensusOrder(),
                                 previousConsensusEvent.getConsensusTimestamp(),
-                                e.toShortString(),
+                                e.getHashedData().getDescriptor(),
```

Comment:
again, we should avoid exposing hashed data

Reply:
Changing to use `GossipEvent`'s `getDescriptor` method